Hello_Repo.cs

```diff
@@ -10,7 +10,7 @@ internal class Program
     {
         static void Main(string[] args)
         {
```

Comment:
Mejor modificar esta parte del código.

```suggestion
        static void Main(string[] args)
        {
```